lib/openstudio-standards/service_water_heating/create_typical.rb

```diff
@@ -0,0 +1,302 @@
+
+          # create service hot water loop for 'One Per Space' and 'One Per Unit' dedicated equipment
+          case loop_type
+          when 'Shared'
+            if is_booster
+              booster_water_use_equipment << water_use_equip
+            else
+              shared_water_use_equipment << water_use_equip
+            end
+          when 'One Per Space', 'One Per Unit'
+            space_water_use_equipment << water_use_equip
+          else
+            OpenStudio.logFree(OpenStudio::Warn, 'openstudio.standards.ServiceWaterHeating', "Water use equipment service loop type #{loop_type} not recognized. Cannot attach equipment to a loop.")
+          end
+        end
+
+        # create water loop for 'One Per Space' and 'One Per Unit' equipment
+        unless space_water_use_equipment.empty?
+          water_heater_capacity_w = num_units * OpenStudio.convert(20.0, 'kBtu/hr', 'W').get
+          water_heater_volume_m3 = num_units * OpenStudio.convert(50.0, 'gal', 'm^3').get
+          num_water_heaters = num_units
+
+          # default to electricity for single units
+          dedicated_water_heating_fuel = water_heating_fuel || 'Electricity'
+
+          # @todo add method to get service water temperature as max of space_water_use_equipment
+          service_water_temperature_c = OpenStudio.convert(140.0, 'F', 'C').get
+
+          # add service water loop with water heater
+          swh_loop = OpenstudioStandards::ServiceWaterHeating.create_service_water_heating_loop(model,
+                                                                                                system_name: "#{space.name} Service Water Loop",
+                                                                                                service_water_temperature: service_water_temperature_c,
+                                                                                                service_water_pump_head: 0.01,
+                                                                                                service_water_pump_motor_efficiency: 1.0,
+                                                                                                water_heater_capacity: water_heater_capacity_w,
+                                                                                                water_heater_volume: water_heater_volume_m3,
+                                                                                                water_heater_fuel: dedicated_water_heating_fuel,
+                                                                                                number_of_water_heaters: num_water_heaters,
+                                                                                                add_piping_losses: true,
+                                                                                                floor_area: OpenStudio.convert(space.floorArea * space.multiplier, 'ft^2', 'm^2').get,
```

Comment:
third time doing this OpenStudio.convert - consider putting into a variable at the start of the `model.getSpaces.each` loop

Reply:
there is actually a unit conversion error here too. fixed.